modules/80_optimization/nash/postsolve.gms

```diff
@@ -283,38 +280,36 @@ loop(trade$(NOT tradeSe(trade)),
 
 *** critertion "infes": and are all solutions optimal?
 loop(regi,
- if((p80_repy(regi,'modelstat') ne 2) and (p80_repy(regi,'modelstat') ne 7),
-     s80_bool = 0;
-     p80_messageShow("infes") = YES;
-  );
-
+     if((p80_repy(regi,'modelstat') ne 2) and (p80_repy(regi,'modelstat') ne 7),
+         s80_bool = 0;
+         p80_messageShow("infes") = YES;
+     );
 *** critertion "nonopt": The next lines are a workaround for the status 7
 *** problem. If the objective value does not differ too much from the last known
 *** optimal solution, accept this solution as if it were optimal. 
-  if (1 le iteration.val,
-    !! no last iteration if this is the first; NA value in p80_repyLastOptim is
-    !! sticky, so test this separately
-    if (   p80_repy(regi,'modelstat') eq 7
+     p80_convNashObjVal_iter(iteration,regi) = abs(p80_repy(regi,'objval') - p80_repyLastOptim(regi,'objval'));
```

Comment:
I would save the `p80_convNashObjVal_iter` parameter without the `abs` transformation, so we can keep the deviation signal in the parameter for further analysis if necessary.

Reply:
changed it, it also had a bug